frontend/mr-admin-flate/src/components/notater/Notat.tsx

```diff
@@ -0,0 +1,52 @@
+import styles from "./Notater.module.scss";
+import { BodyLong, BodyShort, Button } from "@navikt/ds-react";
+import { formaterDatoTid } from "../../utils/Utils";
+import { TrashIcon } from "@navikt/aksel-icons";
+import Lenke from "mulighetsrommet-veileder-flate/src/components/lenke/Lenke";
+import { AvtaleNotat } from "mulighetsrommet-api-client";
+import { useHentAnsatt } from "../../api/ansatt/useHentAnsatt";
+
+interface NotatkortProps {
+  notat: AvtaleNotat;
+  handleSlett: (id: string) => void;
+}
+
+export function Notat({ handleSlett, notat }: NotatkortProps) {
+  const { data: bruker } = useHentAnsatt();
+
+  const lagtTilAv = (notat: AvtaleNotat) => {
+    return (
+      <div className={styles.notatinformasjon_bruker}>
+        <BodyShort>Lagt til av: </BodyShort>
+        {notat.opprettetAv.navIdent === bruker!.navIdent ? (
+          <span className={styles.notatinformasjon_egen_bruker}>
+            <BodyShort className={styles.notatinformasjon_navn}>
+              {notat.opprettetAv.navn}
+            </BodyShort>
+            <Button variant="tertiary" onClick={() => handleSlett(notat.id)}>
+              <TrashIcon fontSize={"1.5rem"} />
+            </Button>
+          </span>
+        ) : (
+          <Lenke
+            to={`https://nom.nav.no/ressurs/${notat.opprettetAv.navIdent}`}
+            target={"_blank"}
+            className={styles.notatinformasjon_navn}
+          >
+            {notat.opprettetAv.navn}
+          </Lenke>
+        )}
+      </div>
```

Comment:
Egentlig kan vel denne forenkles og eneste forskjell er at hvis du har lagt til notatet selv så får du knapp for å slette, hvis ikke så er alt likt. 

Er vel eegentlig ingen god grunn til at du IKKE skal ha lenke hvis du har lagt til notatet selv?